src/js/components/FormField/__tests__/FormField-test.js

```diff
@@ -447,4 +447,27 @@ describe('FormField', () => {
 
     expect(container.firstChild).toMatchSnapshot();
   });
+
+  test('validation', () => {
+    const { container } = render(
+      <Grommet>
+        <Form validate="change">
+          <FormField
+            label="label"
+            validation={{ length: { max: 10, threshold: 0.5 } }}
+            name="issue-description"
+            htmlFor="issue-description"
+          >
+            <TextInput
+              id="issue-description"
+              name="issue-description"
+              placeholder="placeholder"
+            />
+          </FormField>
+        </Form>
+      </Grommet>,
+    );
+
+    expect(container.firstChild).toMatchSnapshot();
```

Comment:
Thanks for getting started on the test.

To improve the test, it might be helpful to:
1. simulate typing something into the input that meets the threshold
2. add an expect statement that checks that the "X characters left" message appears
3. simulate typing more into the input to the max is exceeded
4. add an expect statement that checks that the "X characters over limit" message appears.